builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -994,8 +1004,15 @@ def loadBuildModules(buildModuleUrl){
 
 
 
-def createWebsite(stackName, resourceGroup, storageName){
-  sh "cd ./app && zip -qr ../content.zip *  && cd .."
+def createWebsite(stackName, resourceGroup, storageName, framework){
```

Comment:
This change was breaking Jenkins as it was zipping including node_modules etc. as it ran out of memory.